stockApp/stockApp/Utility/HelperFunctions.swift

```diff
@@ -13,15 +13,28 @@ class HelperFunctions {
     
     class func showValidationsAlertMessage(title: String = Strings.emptyString, message: String = Strings.emptyString, viewController: UIViewController) {
         DispatchQueue.main.async {
-          let alertMessage = UIAlertController(title: title, message: message, preferredStyle: .alert)
-          alertMessage.addAction(UIAlertAction(title: Strings.cancelMessage, style: .default, handler: nil))
-          alertMessage.addAction(UIAlertAction(title: Strings.okMessage, style: .default, handler: {_ in
-            (viewController as? FormViewController)?.coordinator?.goToPreviousViewController()
-            alertMessage.dismiss(animated: false, completion: nil)
-            viewController.dismiss(animated: false, completion: nil)
-          }))
-          alertMessage.view.tintColor = Color.tundoraBlack
-          viewController.present(alertMessage, animated: true, completion: nil)
+            let alertMessage = UIAlertController(title: title, message: message, preferredStyle: .alert)
+            alertMessage.addAction(UIAlertAction(title: Strings.cancelMessage, style: .default, handler: nil))
```

Comment:
```suggestion
            alertMessage.addAction(UIAlertAction(title: Strings.cancelMessage.localized, style: .default, handler: nil))
```

and then use your previous string extension